### PR TITLE
Ruairi's Travel Recommendation

### PR DESCRIPTION
**Pull request by PyrgosRuairi**
https://github.com/UCL-COMP0233-24-25/RSE-Classwork/issues/3

### CODE REVIEW

**Review by r4b61t — APPROVED** [review]
Looks good. Approved

File: europe/README.md [hunk 1]
```diff
@@ -1 +1,3 @@
 # EUROPE
+
+[Cyprus](./cyprus/)
```
Comment (dpshelio) [inline europe/README.md:3]: Link needs to point to the readme file, not the folder.

**Review by dpshelio — CHANGES_REQUESTED** [review]
Thanks for your contribution. Fix the link that's making the checker to fail so it can be merged.

**Review by dpshelio — APPROVED** [review]
:tada:

**Pull request merged**